lib/spandex_datadog/api_server.ex

```diff
@@ -186,11 +189,13 @@ defmodule SpandexDatadog.ApiServer do
 
   @spec send_and_log([Trace.t()], State.t()) :: :ok
   def send_and_log(traces, %{verbose?: verbose?} = state) do
+    headers = @headers ++ [{"X-Datadog-Trace-Count", length(traces)}]
```

Comment:
We might want to convert the count to a string here, just in case. Unless that causes any kind of issues.

Reply:
HTTPoison seems to be fine with it being an integer and forwards it correctly. :man_shrugging: